05-data-structures-part2.Rmd

```diff
@@ -1,245 +1,132 @@
 ---
 layout: page
 title: R for reproducible scientific analysis
-subtitle: Data frames and reading in data
+subtitle: Exploring Data Frames
 minutes: 45
 ---
 
 ```{r, include=FALSE}
 source("tools/chunk-options.R")
-# Silently load in the data so the rest of the lesson works
-library(gapminder)
-#gapminder <- read.csv("data/gapminder-FiveYearData.csv", header=TRUE)
 ```
 
 > ## Learning Objectives {.objectives}
 >
-> * Become familiar with data frames
-> * Be able to read regular data into R
+> - To learn how to manipulate a data.frame in memory
+> - To tour some best practices of exploring and understanding a data.frame when it is first loaded.
 >
 
-## Data frames
+At this ponit, you've see it all - in the last lesson, we toured all the basic data types and data structures in R. Everything you do will be a manipulation of those tools. But a whole lot of the time, the star of the show is going to be the data.frame - that table that we started with that information from a CSV gets dumped into when we load it. In this lesson, we'll learn a few more things about working with data.frame.
 
-Data frames are similar to matrices, except each column can be a
-different atomic type. A data frames is the standard structure for
-storing and manipulating rectangular data sets.  Underneath the hood,
-data frames are really lists, where each element is an atomic vector,
-with the added restriction that they're all the same length.  As you
-will see, if we pull out one column of a data frame,we will have a
-vector. You will probably find that data frames are more complicated
-than vectors and other data structures we have considered, but they
-provide powerful capabilities.
-
-
-Data frames can be created manually with the `data.frame` function:
+We learned last time that the columns in a data.frame were vectors, so that our data is reliable and consistent throughout the column. As such, if we want to add a new column, we need to start by making a new vector:
```

Comment:
pedantic grammar Q: data is vs. data are vs data set is

Is there a SWC position on this issue?